lib/DBDefs/Default.pm

```diff
@@ -52,7 +52,7 @@ sub REPLICATION_ACCESS_TOKEN { '' }
 # On RT_MASTER, setting this to 1 generates '-v2' packets from dbmirror2
 # (in addition to normal dbmirror packets). On RT_SLAVE, downloads the v2
```

Comment:
If dbmirror2 is going to be the "normal" going forward, maybe we should change this comment :)